pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -176,7 +176,7 @@ func newLighthouseAgent(cr *submarinerv1alpha1.ServiceDiscovery) *appsv1.Deploym
 						},
 					},
 
-					ServiceAccountName:            "submariner-operator",
+					ServiceAccountName:            "submariner-lighthouse",
```

Comment:
Nice, we should do more of this for the other services, scoping to their specific permissions/service account :+1: